packages/instantsearch-ui-components/src/components/FrequentlyBoughtTogether.tsx

```diff
@@ -0,0 +1,123 @@
+/** @jsx createElement */
+
+import { cx } from '../lib';
+
+import { createDefaultHeaderComponent } from './recommend-inner/DefaultHeader';
+import { createListViewComponent } from './recommend-inner/ListView';
+
+import type {
+  ItemComponentProps,
+  RecommendClassNames,
+  RecommendTranslations,
+  Renderer,
+} from '../types';
+import type { ViewProps } from './recommend-inner/ListView';
+
+// types
```

Comment:
There are several local types you've migrated and that are no longer used, but you're not getting type hinting because they're currently exported.